manifests/init.pp

```diff
@@ -98,7 +98,7 @@
   Integer[0]                     $systemd_limit_processes         = 64,
   Boolean                        $systemd_private_devices         = true,
   Optional[String[1]]            $systemd_capability_bounding_set = undef,
-  Optional[String[1]]            $systemd_ambient_capabilities    = undef,
+  String[1]                      $systemd_ambient_capabilities    = 'CAP_NET_BIND_SERVICE',
```

Comment:
Not sure changing this default is the best because we cannot reset it, but as the module run the service with an unprivileged user that feels better that switching to running the service as root.

Any thought?